mappings/net/minecraft/client/font/GlyphRenderer.mapping

```diff
@@ -23,6 +36,8 @@ CLASS net/minecraft/class_382 net/minecraft/client/font/GlyphRenderer
 		ARG 2 matrix
 		ARG 3 vertexConsumer
 		ARG 4 light
+	METHOD method_24045 getLayer (Z)Lnet/minecraft/class_1921;
+		ARG 1 seeThrough
```

Comment:
`transparent`

Reply:
This would make sense if `seeThrough` had translucency enabled and the normal text layer didn't (in reality both do). However, the real difference is that `seeThrough` has depth test enabled, which implies that it's meant to be used in a 3D scene (as opposed to a 2D gui). The current name for this bool is purely naming related (ie. "give me the `see_through` layer instead of the normal one). If we want to redefine "seeThrough", it should in my opinion happen in a pure refactor PR that changes **all** instances of this naming scheme, not just in select places.